### ISSUE TITLE
Option to pass in name of environment variable to use

### ISSUE THREAD

**Issue by clbarnes** [kevinburke/sshpass#9]
Sometimes sshpass needs to be used multiple times in the same environment (e.g. jump hosts with different passwords which ban key auth). Ideally `-e "MY_PASS_ENV"` would allow the selection of which variable you want., but some other letter would be fine.

**Comment by camAtGitHub** [comment]
$ sshpass -h

Usage: sshpass [-f|-d|-p|-e] [-hV] command parameters
   -f filename   Take password to use from file
   -d number     Use number as file descriptor for getting password
   -p password   Provide password as argument (security unwise)
   **-e            Password is passed as env-var "SSHPASS"**
   With no parameters - password will be taken from stdin

   -P prompt     Which string should sshpass search for to detect a password prompt
   -v            Be verbose about what you're doing
   -h            Show help (this screen)
   -V            Print version information
At most one of -f, -d, -p or -e should be used

$ sshpass -V

sshpass 1.06

https://github.com/kevinburke/sshpass/blob/master/ChangeLog:
Version 0.02	* Add the "-e" option.

**Comment by clbarnes** [comment]
Right. Sorry if I was unclear; my feature request was to be able to specify the name of the environment variable to be read, so that you can store two passwords in two environment variables and invoke sshpass twice, once for each password. `-f` allows you to specify the name of the file rather than automatically pointing to a hardcoded file; doing the same thing with environment variables would be helpful.

**Comment by clbarnes** [comment]
There's a patch on sourceforge [here](https://sourceforge.net/p/sshpass/patches/14/), for reference.

**Comment by kevinburke** [comment]
Just want to be clear, this project is a mirror of source code I found elsewhere and I have no interest in accepting or adding patches. If you're interested in this I highly encourage you to fork this, I'm happy to link to your project.

**Comment by clbarnes** [comment]
OK, I just assumed that as the owner of a source repo for a project, you might be interested in its development. Having raised the discussion here, I thought I'd also link the outcome of the discussion here in case anyone else came across it in future, as GitHub is much more discoverable than the original source for the average modern open source dev. I'm not asking for any action from you, but I had assumed you were a contributor to the original, in which case your input would have been welcome, as clearly my opening description was not clear enough as to the feature I was suggesting.

**Comment by ling-zhou** [comment]
another version you can try:
https://github.com/ling-zhou/rust-sshpass
